docs/ref/templates/builtins.txt

```diff
@@ -2628,3 +2628,26 @@ template variable with the media prefix :setting:`MEDIA_URL`, e.g.::
 
 By storing the value in a data attribute, we ensure it's escaped appropriately
 if we want to use it in a JavaScript context.
+
+Template tag decorator reference
```

Comment:
I'm not sure this needs to go here. In the _Writing filters_ docs, sure, but this is the ref for the built-in tags and filters.

Plus it's just a copy-and-paste of the text added to the other docs. I'm not convinced that adds much.